API_Guidelines.md

```diff
@@ -102,6 +103,37 @@ Payload
 }
 ```
 
+### GET single record using a query
+#### Request
+URL
+:   `apiHost.com/movies?title=Goodfellas&year=1980`
+
+Ember Data method
+:   `queryRecord('movie', {
+		title: 'Goodfellas',
+		year: '1980' })`
+
+### Why?
+
+> To get a single record based on parameter criteria.
+
+### Response
```

Comment:
response should be an array: https://guides.emberjs.com/v2.11.0/models/finding-records/